### ISSUE TITLE
`ExternalScriptsHandle` does not attach custom attributes to link

### ISSUE THREAD

**Issue by IgnisDa** [sergiodxa/remix-utils#309]
### Describe the bug

Here is my handle function:

```tsx
export const handle: ExternalScriptsHandle<SerializeFrom<typeof loader>> = {
	scripts({ data }) {
		const scripts = []
		scripts.push({
			src: "https://us.umami.is/script.js",
			defer: true,
			"website-id": "abcd",
			"data-domains": undefined
	        });
		return scripts;
	},
};
```

However, this is rendered as:
```html
<script src="https://us.umami.is/script.js" defer="" async=""></script>
```

### Your Example Website or App

https://github.com/IgnisDa/ryot/blob/8204c2f5bc9de32e4181cf60a117db184b9eba88/apps/frontend/app/routes/_dashboard.tsx#L148

### Steps to Reproduce the Bug or Issue

- Use the above handler function anywhere.

### Expected behavior

I expect it to be rendered as:
```html
<script src="https://us.umami.is/script.js" defer="" async="" data-website-id="abcd"></script>
```

### Platform

- OS: All
- Browser: All
- Version: `remix-utils@7.5.0`

**Comment by IgnisDa** [comment]
I think this can be fixed by attaching rest props for this" https://github.com/sergiodxa/remix-utils/blob/e8a199133914a75fb4099e4b2356e73f65380e30/src/react/external-scripts.tsx#L229

**Comment by gar1t** [comment]
Just a bump to this - not supporting `data-xxx` breaks this abstraction.

**Comment by wouterds** [comment]
Would be nice, e.g. the Cloudflare beacon uses this custom `data-cf-beacon` attribute. You can have it automagically insert but then the console is full of hydration mismatch errors.

**Comment by Rocinante89** [comment]
Agreed with @gar1t on this one. Came across this issue integrating Cloudflare beacon.

**Comment by theosenoussaoui** [comment]
Would love to the possibility of passing custom attributes as well !

**Comment by theocerutti** [comment]
same!
